evap/staff/importers/base.py

```diff
@@ -56,11 +62,29 @@ class Category(Enum):
     message: str
 
 
+@dataclass
+class ImporterDecisionEntry(ImporterLogEntry):
```

Comment:
Not a big fan of having this in the log, I think the log and the decisions are semantically different. If we have both in the same structure, we should rename it, as it is no more a "log" of things that happened.